src/neo/Consensus/ConsensusContext.cs

```diff
@@ -422,10 +457,16 @@ public void Serialize(BinaryWriter writer)
             writer.Write(ViewNumber);
             writer.Write(TransactionHashes ?? new UInt256[0]);
             writer.Write(Transactions?.Values.ToArray() ?? new Transaction[0]);
+
             writer.WriteNullableArray(PreparationPayloads);
             writer.WriteNullableArray(CommitPayloads);
             writer.WriteNullableArray(ChangeViewPayloads);
             writer.WriteNullableArray(LastChangeViewPayloads);
+
+            writer.WriteNullableArray(FuturePreparationPayloads);
+            writer.WriteNullableArray(FutureCommitPayloads);
+            writer.WriteNullableArray(FutureChangeViewPayloads);
+            writer.WriteNullableArray(FutureRecoveryPayloads);
```

Comment:
I think we don't have to save the future payloads.

Reply:
I do not know. When is this used?

Reply:
It is used when sending `Commit` payload.

Reply:
I did not understand, this is the `ConsensusContext`, `Commit` Serialize a `ConsensusMessage`.

I think this is called on `public bool Load()` only, @erikzhang.
In this sense, it is ok to leave it here.

Reply:
`Load()` reads the data. When sending `Commit`, it save the data.

Reply:
That is true.
Do you think it is too much time consuming?
Those array all together are `4 x Validator.Length x ConsensusPayload.size`.

So, let's delete this.

Reply:
Perhaps the benefits of writing them is not so good, since a Recovery would arrive anyway...

Reply:
Done.

Reply:
@erikzhang, 
After deleting that from Serialization/Deserialization the tests still pass on `TestSerializeAndDeserializeConsensusContext()`.

In this sense, the following line is expected to still work, right?
`var copiedContext = TestUtils.CopyMsgBySerialization(consensusContext, new ConsensusContext(null, null));`